openstates/mn/__init__.py

```diff
@@ -161,23 +120,29 @@
         '79th Legislature, 1995-1996',
         '79th Legislature, 1995 1st Special Session',
         '89th Legislature, 2015-2016',
-    ],
-    'feature_flags': ['subjects', 'events', 'influenceexplorer'],
-}
+    ]
+
+    def get_organizations(self):
+        legis = Organization('Minnesota Legislature', classification='legislature')
 
+        upper = Organization('Minnesota Senate', classification='upper',
+                             parent_id=legis._id)
+        lower = Organization('Minnesota House of Representatives',
+                             classification='lower', parent_id=legis._id)
 
-def session_list():
-    from billy.scrape.utils import url_xpath
-    return url_xpath('https://www.revisor.mn.gov/revisor/pages/search_status/'
-                     'status_search.php?body=House',
-                     '//select[@name="session"]/option/text()')
+        for n in range(1, 68):
+            upper.add_post(label=str(n), role='Senator',
+                           division_id='ocd-division/country:us/state:mn/sldu:{}'.format(n))
+            lower.add_post(label=str(n) + 'a', role='Representative',
```

Comment:
Okay, did that. For clarity, we're leaving the ocd division alone, right? i.e. `ocd-division/country:us/state:mn/sldl:1a`